mitiq/pec/pec.py

```diff
@@ -89,11 +89,6 @@ def execute_with_pec(
             mean squared deviation of the sampled values from 'pec_value'.
             This is returned only if ``full_output`` is ``True``.
 
-    .. [Temme2017] : Kristan Temme, Sergey Bravyi, Jay M. Gambetta,
```

Comment:
Yeah, sphinx was throwing a warning because it was doubly defined.

Reply:
I think it is not nice to use some global references (e.g. Temme...) and some local references (e.g. Endo...) in the docstring of the function.
What about if we:

1. Keep as before but rename local references in function doc-strings with arbitrary names to avoid conflicts.
2. Move all the local references used in our functions to the global page.

Reply:
(2) sounds much better to me.

Reply:
Actually, I don't think both local and global refs is bad. Other thoughts?